db/migrations/20210111183822_get_max_block_height_of_transformed_diffs.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+
+CREATE OR REPLACE FUNCTION api.get_max_block_height_of_transformed_diffs()
```

Comment:
Now I'm just being picky but I wonder if I could convince you to rename this to `get_max_transformed_diff_block`, and then I can update #420 to be `get_max_transformed_event_block` and they'll look similar (reminder to update the migration and test names if you make the change)